data/src/main/java/io/gnosis/data/models/Transaction.kt

```diff
@@ -0,0 +1,11 @@
+package io.gnosis.data.models
+
+sealed class Transaction(open val nonce: String) {
+
+    data class ChangeMastercopy(override val nonce: String) : Transaction(nonce)
```

Comment:
I put that just because data classes need at least one member variable/value. `nonce` seemed like a good candidate as all the subtypes will have it for sure.